design/README.md

```diff
@@ -0,0 +1,73 @@
+# Code specification
+
+## Motivation
+
+Replace the current APFELcomb code with a simpler and faster tool to generate FK Tables for NNPDF fits.
+
+## APFELcomb design and issues
+
+### Workflow
+
+The apfelcomb project compiles a program which connects several tools and
+databases from NNPDF and external providers in order to generate FK Tables,
+where observables are stored into tables with a predefined input scale energy value for PDFs evaluation.
+
+The user/develop should include the custom implementation of the combination operation for each specific process category. All operations involving DGLAP, DIS and Fixed-target DY are performed at runtime. Operations involving the generation of hadronic observables with or without electroweak corrections are performed off-line and stored on git-lfs repositories.
```

Comment:
Sorry for nitpicking, but since this is a specs document: are you sure that you would like to rely on using `git-lfs` always?
(of course the problem it's not the document itself, but the code written after it)

Reply:
I think we can/should have both: git-lfs and server. In particular the server could render the content of the git-lfs repos for fktables, evolution kernels and pineappl grids.

The server is good because:
- easy & fast access
- public
- already implemented with backups

The repo is good because:
- we can check the revision if bugs are found
- is an extra source of backup

Reply:
That's fine, we noticed that dealing with `git-lfs` on the cluster it may be harder because you need to install an extra client, and you may not have admin privileges and so on and so forth.

If the `git` repo is used just for the generation (man-work) and then they are mirrored on the server it would be perfectly fine.